src/button/button.css

```diff
@@ -1,93 +1,204 @@
-/* "Unstyled" button css taken from https://gist.github.com/MoOx/9137295 */
```

Comment:
Unstyled buttons are important. We should keep them. See explanation here: https://github.com/openmrs/openmrs-esm-styleguide/blob/03ff09f407f8a1e0e9e607a6351ed096d97d320b/src/button/button.stories.html#L12 and https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--buttons